frontend/src/components/match/MatchCard.js

```diff
@@ -103,6 +104,12 @@ const TournamentMatch = props => {
       userAccessQuery?.data?.admin_team_ids.indexOf(props.match["team_2"].id) >
         -1);
 
+  const isCurrentTeamMatchAdmin = () =>
+    userAccessQuery?.data?.admin_team_ids?.length > 0 &&
+    userAccessQuery?.data?.admin_team_ids.indexOf(
+      props.match[`team_${currTeamNo()}`].id
+    ) > -1;
+
```

Comment:
checking whether user is admin of current team only. When we check for both teams like in `isMatchTeamAdmin`, the submit buttons are shown in both teams matches

Reply:
ignore... forgot that we need to show the buttons on schedule page match cards also...